public/locales/en-US/translations.json

```
@@ -37,7 +37,15 @@
   "24H": "24H",
   "30D": "30D",
   "total_transactions": "# of Txns",
+  "successful_transactions": "# of Sucessful txns",
```

```suggestion
  "successful_transactions": "# of Successful txns",
```
Please be consistent with capitalization